templates/templates/base.html

```diff
@@ -46,13 +52,19 @@
   <!-- footer content goes here -->
   {% include "templates/footer.html" %}
 
-  <!-- baidu -->
+  <!-- baidu 
   <script>
     var _bdhmProtocol = (("https:" == document.location.protocol) ? " https://" : " http://");
     document.write(unescape("%3Cscript src='" + _bdhmProtocol + "hm.baidu.com/h.js%3F4ee06d994b22c33cd3732c3e69c97b8f' type='text/javascript'%3E%3C/script%3E"));
   </script>
 
+  <script>
+    ubuntu.globalNav.setup();
+  </script> -->
```

Comment:
Can you remove these commented out lines please

Reply:
Done